google-style.yml

```diff
@@ -516,6 +582,9 @@ Style/RescueModifier:
 Style/ReturnNilInPredicateMethodDefinition:
   Enabled: true
 
+Style/SafeNavigationChainLength: # new in 1.68
```

Comment:
Can you provide a short note explaining why we want to disable this one?

Reply:
added